data/notebooks/requirements.txt

```diff
@@ -0,0 +1,40 @@
+
+
+# configuration requirements
+
+python-dotenv>=0.5.1
+
+# external tools requirements
+tqdm
```

Comment:
every time someone adds a dependency on an external package without specifying a version, god kills a puppy :P

In all seriousness, this is a recipe for disaster. If one of these packages releases a major version in the future, with a full refactor of their API, your whole application will stop working, and you'll have no control over it.

Reply:
I've already added the versions to all the packages